sympy/physics/continuum_mechanics/beam.py

```diff
@@ -290,6 +290,12 @@ def join(self, beam, via="fixed"):
             new_beam._hinge_position = self.length
             return new_beam
 
+    def polar_moment(self,axis=None):
+        if(axis==None or axis=='x'):
```

Comment:
No need for brackets.

It may make sense to raise an exception if this does not hold.